spec/rubocop/cop/i18n/gettext/decorate_string_spec.rb

```diff
@@ -5,18 +5,66 @@
 describe RuboCop::Cop::I18n::GetText::DecorateString do
   let(:config) { RuboCop::Config.new }
   subject(:cop) { described_class.new(config) }
+  before(:each) do
+    investigate(cop, source)
+  end
+
+  context 'decoration need for string' do
+    it_behaves_like 'a_detecting_cop', 'a = "A sentence that is not decorated."', '_', 'decorator is missing around string'
+    it_behaves_like 'a_detecting_cop', 'thing("A sentence that is not decorated.")', '_', 'decorator is missing around string'
+    it_behaves_like 'a_detecting_cop', 'thing("A sentence that " + "is not decorated.")', '_', 'decorator is missing around string'
```

Comment:
Originally, this was written like:
```
it_behaves_like 'a_detecting_cop', "thing(\"A sentence that " + "is not decorated.\")", '_', 'decorator is missing around string'
```
But I'm not sure that was testing the correct thing, as it would have been parsed as `thing("A sentence that  + is not decorated.")`?